src/main/java/nextstep/session/domain/Money.java

```diff
@@ -1,16 +1,18 @@
-package nextstep.enrollment.domain;
+package nextstep.session.domain;
 
-import nextstep.exception.MoneyIllegalArgumentException;
+import java.util.Currency;
+
+import nextstep.session.exception.MoneyIllegalArgumentException;
 
 public class Money {
     private final long amount;
-    private final String currency;
+    private final Currency currency;
 
     public Money(long amount) {
-        this(amount, "KRW");
+        this(amount, Currency.getInstance("KRW"));
```

Comment:
"KRW" 문자가 더 명확하게 읽히지만
상수 사용이 더 나을 것 같아서요!

```suggestion
        this(amount, Currency.getInstance(Locale.KOREA));
```

Reply:
@nooose 
동의합니다 ㅎㅎ 수정하도록 하겠습니다